binding/android/settings.gradle

```diff
@@ -0,0 +1,2 @@
+rootProject.name = "Koala"
```

Comment:
`libpv_koala.so` and `koala_params.pv` can be ignored and added to the gradle build. That way we don't need to remember to update them.